Updated environment file, updates to scripts, added README.md (#12)
WayScience/gene_dependency_representations: 0.data-download/README.md

> ### Description of the Data ###

Reviewer: ```suggestion
# Description of the Data
```

We typically start `.md` files with something called an `H1` header (1 #)

> # DepMap Public 22Q4

Reviewer: ```suggestion
## DepMap Public 22Q4
```

This is followed by an `H2` header. (Two #'s)

This DepMap release contains data from in vitro studies of genetic dependencies in cancer cell lines using CRISPR/Cas9 loss-of-function screens from project Achilles, as well as genomic characterization data from the CCLE project.

Data resource:
https://depmap.org/portal/download/all/

> 
> 
> 

Reviewer: ```suggestion
```

It is safe to delete excess new lines. We typically like to keep `.md` files clean and consistently formatted.

> ### Files used in the WayScience Project ###

Reviewer: ```suggestion
## Files used in our multivariate gene dependency project
```

To be more descriptive :) (also switching to `H2` header)

> ### CRISPR_gene_dependency.csv ###

Reviewer: ```suggestion
### CRISPR_gene_dependency.csv
```

An `H3` header works nicely here!

> This data comprises scores from individual CRISPR gene knockout screens in cancer cell lines for every gene across many different ages and cancer types. The values represent gene dependency probability estimates for cell survival and growth for all models in the integrated gene effect. The higher the score, the greater the gene dependency. 

Reviewer: ```suggestion
This data comprises scores from individual CRISPR gene knockout screens in cancer cell lines for every gene across many different ages and cancer types.
The values represent gene dependency probability estimates for cell survival and growth for all models in the integrated gene effect.
The higher the score, the greater the gene dependency. 
```

We follow the [one-sentence-per-line convention](https://news.ycombinator.com/item?id=31808093) in markdown files.

> This information is derived from the CRISPRGeneEffect estimates for all models which were introduced to an algorithm, Chronos, to infer gene knockout fitness effects based on an explicit model of cell proliferation dynamics after CRISPR gene knockout.

Reviewer: ```suggestion
The probability estimate is derived from the CRISPRGeneEffect estimates for all models which were introduced to an algorithm, Chronos, to infer gene knockout fitness effects based on an explicit model of cell proliferation dynamics after CRISPR gene knockout.
```

Is my edit correct? I'd like to clarify what "this" means

> 

Reviewer: ```suggestion
See the following resource for more information: https://forum.depmap.org/t/depmap-genetic-dependencies-faq/131
```

Columns: gene

> Rows: the ModelID

Reviewer: How many columns (genes) and rows (cell line IDs)?

> 

Reviewer: ```suggestion
```

Delete excess line

> ### sample_info.csv ###

Reviewer: ```suggestion
### sample_info.csv
```

This file gives detailed information on the cell lines, type of cancer, sex, and age of the patient.

> Columns: description of cancer, and patient information

Reviewer: What type of patient information?

## Files used in our multivariate gene dependency project

> ### CRISPRGeneDependency.csv

Author: Updated file name based on new data set (22Q4)

Reviewer: oh that's annoying that the file name changed! oh well, definitely a good thing for you to notice!!

This data comprises scores from individual CRISPR gene knockout screens in cancer cell lines for every gene across many different ages and cancer types.
The values represent gene dependency probability estimates for cell survival and growth for all models in the integrated gene effect.
The higher the score, the greater the gene dependency. 

The probability estimate is derived from the CRISPRGeneEffect estimates for all models which were introduced to an algorithm, Chronos, to infer gene knockout fitness effects based on an explicit model of cell proliferation dynamics after CRISPR gene knockout.

Columns (18,333): gene

Rows (1,087): the ModelID

> ### Model.csv

Author: Updated file name based on new data set (22Q4)

Reviewer: Did the row and column counts change?

Author: Ah, yes they did. I need to update the row and column numbers from both files, thanks!

The probability estimate is derived from the CRISPRGeneEffect estimates for all models which were introduced to an algorithm, Chronos, to infer gene knockout fitness effects based on an explicit model of cell proliferation dynamics after CRISPR gene knockout.

> Columns (17,387): gene

Author: Used pandas shape to get number
